.markdownlint.yaml

```diff
@@ -0,0 +1,16 @@
+# Unordered list symbol
```

Comment:
Do you need this file if you are going to pull the .markdownlint.yaml from main? Or we could just commit the same version of .markdownlint.yaml in the root directory in all the branches -- that might be simpler.

Reply:
`validate-markdown.yaml` pulls `package.json` and `package-lock.json` from `main`, most likely to make sure that the same version of mdv is used across all branches, so it makes sense to also use the same linter ruleset across all branches.

The "branch-local" copy of .markdownlint.yaml allows formatting the current spec markdown locally.